testing/perl-crypt-openssl-x509/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Automatically generated by apkbuild-cpan, template 2
+# Contributor: Timothy Legge <timlegge@gmail.com>
+# Maintainer: Timothy Legge <timlegge@gmail.com>
+pkgname=perl-crypt-openssl-x509
+_pkgreal=Crypt-OpenSSL-X509
+pkgver=1.812
+pkgrel=0
+pkgdesc="Perl extension to OpenSSL's X509 API."
+url="https://metacpan.org/release/Crypt-OpenSSL-X509/"
+arch="all"
+license="GPL PerlArtistic"
+cpandepends=""
+cpanmakedepends="openssl-dev"
+cpancheckdepends=""
+depends="$cpandepends"
+makedepends="perl-dev $cpanmakedepends"
+checkdepends="$cpancheckdepends"
+subpackages="$pkgname-doc"
+source="https://cpan.metacpan.org/authors/id/J/JO/JONASBN/Crypt-OpenSSL-X509-$pkgver.tar.gz"
+builddir="$srcdir/$_pkgreal-$pkgver"
+
+build() {
+	cd "$builddir"
```

Comment:
These can be removed as well, as abuild is now taking care of it.

Reply:
Done, will address the others.  I seem to have remembered that part way through the PRs